.github/workflows/integration-tests.yml

```
@@ -0,0 +1,42 @@
+        uses: DeLaGuardo/setup-clojure@13.1
+        with:
+          cli: "1.11.1.1139"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y jq lsof
+
+      - name: Cache dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)